packages/core/src/entities/value.ts

```diff
@@ -0,0 +1,31 @@
+import type { RuntimeEngine } from '../runtime-engine';
+import { CREATE_RUNTIME, REGISTER_VALUE } from '../symbols';
+import type { EnvVisibility } from '../types';
+import { FeatureOutput } from './output';
+
+export class Value<T, ProvidedFrom extends EnvVisibility> extends FeatureOutput<
```

Comment:
I think it requires some explanation to why, and when, we want to use thios entity.
If I understand every service which in not .allowRemoteAccess() is Value?

Reply:
It's kind of true but not a requirement. I wanted that Service will inherit form this but it creates more complex implementation. The Service class should be part of the communication when we extract it. Value is more basic.

Reply:
I don't think we need to document it right since we don't have docs for the other entities.

I want to document everything soon.